File: fizzbuzz.php
#!/usr/bin/env php
<?php

for ($n = 1; $n <= 100; $n++) {
  $output = null;
  $output .= ($n % 3 == 0 ? 'Fizz' : null);
  $output .= ($n % 5 == 0 ? 'Buzz' : null);
  echo ($output ?: $n)."\n";
}

?>

File: fizzbuzz.pl
#!/usr/bin/env perl

#for (my $n = 1; $n <= 100; $n++) {
for my $n (1..100) {
  my $output = "";

  if ($n % 3 eq 0) { $output .= 'Fizz'; }
  if ($n % 5 eq 0) { $output .= 'Buzz'; }
  if ($output) { print "$output"; } else { print "$n"; }
  print "\n";
}
